src/Avalonia.Controls/Navigation/Frame.cs

```diff
@@ -0,0 +1,290 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Avalonia.Controls {
+
+    /// <summary>
+    /// Control allow execute navigation wihtin itself.
+    /// </summary>
+    public class Frame : Panel
+    {
```

Comment:
Why did you choose a Panel? I dont think a Frame will have multiple childs. In my opinion ContentControl would be the better fit.